src/alchemtest/gmx/expanded_ensemble/case_3/descr.rst

```diff
@@ -0,0 +1,22 @@
+Gromacs: Host CB7 and Guest C3 in water
+=========================
```

Comment:
Please extend the `====` line to be at least as long as the text. This must be legal [restructured text section header](http://www.sphinx-doc.org/en/stable/rest.html#sections) to be correctly type set as documentation.